frontend/src/pages/Browse.tsx

```diff
@@ -43,6 +45,8 @@ const Browse = (props: any) => {
         ? <div className={classes.progressCircle}><CircularProgress /></div>
         : <></>}
       <Results />
+      <LoginModal />
```

Comment:
Does it just exist in every single page where it can popup now? Is there no way to have it popup anywhere?

Reply:
that seems to be the case. we could put them somewhere higher up like app.tsx if you want it to appear everywhere so that's worth considering. what do you think?

Reply:
Yeah, probably. Need to figure out where that will go

Reply:
we should discuss this, what do you think is best?

also i don't think it's a bad idea to have the modals be page specific. if you think about it you can have more control over when modals should or should not appear, and this control is incredibly easy to implement (simply add the modal component in the page in which it should exist)